intranet/apps/polls/models.py

```diff
@@ -94,6 +94,23 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        users = []
+        if self.groups.all():
```

Comment:
```suggestion
        if self.groups.exists():
```

`exists()` checks if the queryset exists (has a non-zero length).